src/commons/BrowserUtils.ts

```diff
@@ -707,8 +707,57 @@ export namespace BrowserUtils {
   }
 
   /**
-   * When switching between iframes, without wait it will fail to switch to iframe
    *
+   * @param mouseButton -  {LEFT = 0, MIDDLE = 1 , RIGHT = 2}
+   */
+  export function mouseButtonDown(mouseButton?: string): void {
```

Comment:
Don't use a default here. If you insist on it - use an explicit [default value](https://www.typescriptlang.org/docs/handbook/functions.html#optional-and-default-parameters)

Reply:
done